api/README.md

```diff
@@ -35,43 +35,15 @@ root
 └── docker-compose.yml  Config file for docker-compose tool, used for local development
 ```
 
-## Information
+## Local Development
+
+See [development.md](./development.md) for installation and development instructions.
```

Comment:
This link doesn't work

```suggestion
See [development.md](../documentation/api/development.md) for installation and development instructions.
```

Reply:
I updated it (didn't see yr commit suggestion for this one), thanks.